app/scripts/onlyKey/OnlyKeyComm.js

```diff
@@ -2040,25 +2040,7 @@ function saveBackupFile(e) {
   var backupData = ui.backupForm.backupData.value.trim();
   if (backupData) {
     d = new Date();
-    dMonth = d.getMonth() + 1;
-    dDate = d.getDate();
-    dYear = d.getFullYear();
-    dHour = d.getHours() + 1 < 12 ? d.getHours() : d.getHours() - 12;
-    dMinutes = (d.getMinutes() < 10 ? "0" : "") + d.getMinutes();
-    dM = d.getHours() + 1 < 12 ? "AM" : "PM";
-    df =
-      dMonth +
-      "-" +
-      dDate +
-      "-" +
-      dYear +
-      "-" +
-      dHour +
-      "-" +
-      dMinutes +
-      "-" +
-      dM;
-    var filename = "onlykey-backup-" + df + ".txt";
+    var filename = "onlykey-backup-" + d.toISOString() + ".txt";
```

Comment:
This would be a good enhancement. While I agree with the idea behind this change, the reason for the existing format is cross-platform support. Unfortunately, ISO dates are not valid filenames in Windows.
![image](https://user-images.githubusercontent.com/601129/120090590-cb832700-c0d1-11eb-8c5f-383ddcc7b913.png)

How about "YYYY-MM-DDTHH-MM.txt"?

Reply:
Makes sense, thanks - see latest commit. I added seconds on the end as well in case two backups are taken close together.